### PR TITLE
Small change to resource manager settings

### LINKED ISSUE #4212: Dropdown list "Mode" of ResourceManager settings is not localizable
## Description of bug
If you want to localize the new ResourceManager module, there are no entries to localize the items of the "Mode" dropdown list ("Normal", "User", "Group"), so they stay in English.

## Steps to reproduce
1. Add a language to your portal, or use a language already added different than English
2. Go to the Language Editor to translate Local Resources/DesktopModules/ResourceManager/App_LocalResources/Settings.ascx
3. Translate the control
4. Go to module setttings of the ResourceManager module, the dropdown list is not translated

## Expected behavior
Translatable items.

## Affected version

* [ ] 10.00.00 alpha build
* [X] 09.08.00 release candidate
* [ ] 09.07.02 latest supported release

## Affected browser

* [X] All browsers, as this is not a browser issue

### PR DESCRIPTION
**Pull request by MaiklT**
Fixes #4212

## Summary

Added keys to resource file, and code to retrieve localized values.

### CODE REVIEW

**Review by david-poindexter — CHANGES_REQUESTED** [review]
@MaiklT you may want to check this one a bit closer.  It looks like perhaps a copy/paste gone wrong with the RESX file.  :)

**Review by valadas — APPROVED** [review]
Looks good to me now

**Comment by valadas** [comment]
I mean, it looked fine but Localization is not existing in context here, probably need to test this one better

File: DNN Platform/Modules/ResourceManager/Settings.ascx.cs [hunk 1]
```diff
@@ -1,75 +1,75 @@
...
+        {
+            try
+            {
+                if (this.Page.IsPostBack)
+                {
+                    return;
+                }
+
+                var displayTypesValues = Enum.GetValues(typeof(Constants.ModuleModes)).Cast<Constants.ModuleModes>();
+                var displayTypes = displayTypesValues.Select(t => new ListItem(Localization.GetString(string.Format("ModuleModes.{0}", Utils.GetEnumDescription(t)), LocalResourceFile), ((int)t).ToString())).ToArray();
```
Comment (bdukes) [inline DNN Platform/Modules/ResourceManager/Settings.ascx.cs:31]:
```suggestion
                var displayTypes = displayTypesValues.Select(t => new ListItem(Localization.GetString($"ModuleModes.{Utils.GetEnumDescription(t)}.Text", LocalResourceFile), ((int)t).ToString())).ToArray();
```

**Comment by mitchelsellers** [comment]
Changes were requested to this PR that have not yet been addressed, we are going to close this PR at this time.

Feel free to correct the requested items and you can re-open or re-submit it.

**Pull request closed without merging**